app/main/forms.py

```diff
@@ -271,14 +271,21 @@ def validate(self):
 
 class EditFrameworkStatusForm(FlaskForm):
 
-    status = DMRadioField(
-        "Framework status",
-        hint="Tell the rest of the team before making a change",
+    status = RadioField(
+        id="input-status-1",  # TODO: change to input-copy_service when on govuk-frontend~3
+        label="Framework status",
+        description="Tell the rest of the team before making a change",
         validators=[
             validators.InputRequired(message='You must choose a framework status')
```

Comment:
To follow [Design System guidance](https://design-system.service.gov.uk/components/radios#if-there-are-more-than-two-options), this should be "Select a framework status"

Reply:
Good catch, thanks. Fixed.